projects/sw-routing/build.js

```diff
@@ -65,6 +70,37 @@ const bundledJSNext = () => {
   });
 };
 
-module.exports = () => {
+/** module.exports = () => {
   return Promise.all([minifiedUMD(), bundledJSNext()]);
+};**/
+
+module.exports = () => {
+  console.log('Here :S');
+  return new Promise((resolve, reject) => {
+    gulp.src([
+      path.join(__dirname, 'src', '**', '*.js'),
+      path.join(__dirname, '..', '..', 'lib', '**', '*.js')
+    ])
+    .pipe(sourcemaps.init())
+    // transform the files here.
+    .pipe(rollup({
+      entry: path.join(__dirname, 'src', 'index.js')
+    }))
+    .pipe(babel({
+      plugins: ['remove-comments'],
+      presets: ['babili']
+    }))
+    .pipe(sourcemaps.write())
+    .pipe(rename(pkg.main))
+    // This is just the UMD model for now.
+    .pipe(gulp.dest(destPath))
+    .on('error', err => {
+      console.error(err);
+      reject(err);
+    })
+    .on('end', () => {
+      console.log('Here ENDED:S');
```

Comment:
Yeah - this is not final by a long shot, just wanted to check on opinion of this approach.